src/main/java/com/gigacal/config/PersistenceConfig.java

```diff
@@ -0,0 +1,7 @@
+package com.gigacal.config;
+
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@EnableJpaRepositories(basePackages = "com.gigacal.repository")
+public class PersistenceConfig {
```

Comment:
Perhaps this class can also be removed - can you check if everything is working as intended without it?